Ska/engarchive/remote_access.py

```diff
@@ -7,16 +7,76 @@
 import sys
 import os
 import getpass
+from pathlib import Path
+
 try:
     import ipyparallel as parallel
 except ImportError:
     from IPython import parallel
 from six.moves import input
 
-# To use remote access, this flag should be set True (it is true by default
-# on Windows systems, but can manually be set to true on Linux systems
-# if you don't have direct access to the archive)
-access_remotely = sys.platform.startswith('win')
+
+def get_data_access_info(is_windows=sys.platform.startswith('win')):
```

Comment:
Factored this logic into a function mostly to allow testing, but it's also a good thing.